content/english/Kusto-KC7/activity-5.md

```diff
@@ -0,0 +1,87 @@
+---
+title: "Bonus 1: Phishy Activity"
+draft: false
+weight: 20
+---
+
+
+## Hackers Sending0 Malware Docs
```

Comment:
```suggestion
## Hackers Sending Malware Docs
```

Reply:
This was the original format for it, some cybersecurity humor. But fair point it doesn't translate well here and just looks like a mistake. I'll take it off